.github/workflows/scylla-rust-ci.yml

```diff
@@ -0,0 +1,33 @@
+name: Scylla Rust CI
+
+on:
```

Comment:
is this the same one as calypso?

Reply:
pretty much.  removed tests bc tests call the db and audit bc we would fail lol.

Reply:
well were gonna have to add tests back once we do them